jpaNew/src/main/java/com/example/jpaNew/repositories/CommentsRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.jpaNew.repositories;
+
+import com.example.jpaNew.entities.Comments;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CommentsRepository extends JpaRepository<Comments, Long> {
+    Comments findByIdGreaterThanEqualAndIdLessThanEqual(Long startId, Long endId);
```

Comment:
How this is logical ? 
How dumb are you thinking! 
startId and endId both provide a range correct!
A range will can be 0 t0 N correct. So how come return be Comment?
Secondly, of all make all methods